src/MuTalk-Utilities/MTMutantOperatorAnalysis.class.st

```diff
@@ -54,18 +46,18 @@ MTMutantOperatorAnalysis >> findOperatorsWithoutMutantsFor: aDictionary [
 		                           aDictionary keys includes: ope ].
 
 	^ Dictionary new
-		  addAll: (operatorsWithoutMutants collect: [ :each | each -> 0 ]) asDictionary;
+		  addAll:
+			  (operatorsWithoutMutants collect: [ :each | each -> 0 ])
+				  asDictionary;
 		  yourself
 ]
 
 { #category : 'computing' }
-MTMutantOperatorAnalysis >> getMutantOperatorsDictionaryFor: aName [
-
-	| analysis dic classesToMutate |
-	classesToMutate := aName asPackage definedClasses.
+MTMutantOperatorAnalysis >> getMutantOperatorsDictionary [
```

Comment:
remember that in Pharo code we don't use the `get/set` prefixes in methods as a convention.

Reply:
Indeed, I will change that.